doc/doxygen/src/Examples.dox

```diff
@@ -174,6 +174,57 @@
  * url = LocatableContentTrait(resolved_asset).getLocation()  # May be None
  * @endcode
  *
+ * @section examples_inspecting_a_reference Inspecting a reference
```

Comment:
[plan] I know we didn't put this in the ACs, but would be nice to extend the publishing/resolve methods docs/docs to indicate that `entityTraits` exists and can be of assistance. Perhaps the glossary entry for trait sets too.

Reply:
Added some cross-referencing in 5c975239 and 8837daae and 1632609c

Reply:
Nice, thanks.